wgsl/index.bs

```diff
@@ -2486,14 +2535,25 @@ pipeline_stage
   </xmp>
 </div>
 
-## Shader Stages TODO ## {#shader-stages}
+The set of <dfn noexport>functions in a shader stage</dfn> is the union of:
+
+* The entry point function for the stage.
+* The targets of function calls from within the body of a function
+    in the shader stage, whether or not that call is executed.
+
+The union is applied recursively, and stabilizes to a finite set.
```

Comment:
"Transitive closure"?

Reply:
Yes, we're generating at transitive closure, but I wasn't sure how to word it as a verb. Also, it's jargon.

How about

> The union is applied repeatedly until it stabilizes.  It will stabilize in a finite number of steps.

Reply:
I've updated with this new wording.